scipy/optimize/tests/test__dual_annealing.py

```diff
@@ -209,7 +209,7 @@ def func(x):
 
     def test_max_fun_ls(self):
         ret = dual_annealing(self.func, self.ld_bounds, maxfun=100,
-                             seed=self.seed)
+                             rng=self.seed)
```

Comment:
Looks like we left this instance of `seed`:

https://github.com/scipy/scipy/blob/1f5c1a83122d42bf8c01296338af4e3e8fadb058/scipy/optimize/tests/test__dual_annealing.py#L119-L122

I'm fine with this as-is. Maybe in the future we should add a comment so it's obvious in the PR diff.